package.json

```diff
@@ -22,15 +22,17 @@
     "tdd": "node scripts/tdd.js",
     "test": "SERVICES=* yarn services && mocha --exit --expose-gc 'packages/dd-trace/test/setup/node.js' 'packages/*/test/**/*.spec.js'",
     "test:core": "mocha --exit --expose-gc --file packages/dd-trace/test/setup/core.js \"packages/dd-trace/test/**/*.spec.js\"",
-    "test:core:ci": "nyc --include \"packages/dd-trace/src/**/*.js\" -- npm run test:core",
+    "test:core:ci": "nyc --include \"packages/dd-trace/src/**/*.js\" -- npm run test:core -- --reporter=mocha-junit-reporter",
     "test:plugins": "mocha --exit --file \"packages/dd-trace/test/setup/core.js\" \"packages/datadog-plugin-@($(echo $PLUGINS))/test/**/*.spec.js\"",
     "test:plugins:ci": "yarn services && nyc --include \"packages/datadog-plugin-@($(echo $PLUGINS))/src/**/*.js\" -- npm run test:plugins",
     "test:integration": "mocha --timeout 30000 \"integration-tests/**/*.spec.js\"",
     "leak:core": "node ./scripts/install_plugin_modules && (cd packages/memwatch && yarn) && NODE_PATH=./packages/memwatch/node_modules node --no-warnings ./node_modules/.bin/tape 'packages/dd-trace/test/leak/**/*.js'",
     "leak:plugins": "yarn services && (cd packages/memwatch && yarn) && NODE_PATH=./packages/memwatch/node_modules node --no-warnings ./node_modules/.bin/tape \"packages/datadog-plugin-@($(echo $PLUGINS))/test/leak.js\"",
     "cover:merge": "nyc merge ./.nyc_output ./.nyc_merge/$CIRCLE_JOB.json",
     "cover:report": "nyc report -t ./.nyc_merge -n 'packages/**/*.js'",
-    "codecov": "codecov"
+    "codecov": "codecov",
+    "junit:upload:ci": "node ./scripts/junit_report.js",
+    "junit:upload": "datadog-ci junit upload --tags runtime.version:$CI_NODE_VERSION --service dd-trace-js-core-tests ./test-results.xml"
```

Comment:
Where does `datadog-ci` come from when running this if it's not in the dependencies?

Reply:
Also, what's the benefit of ever running this locally? Is there a way to differentiate between local and CI runs?

Reply:
yeah I'll just remove this and run it on the `junit_report` script file